File: API_MyWallet/app/src/main/java/service/BancoService.java
package service;

import java.util.List;

import dao.BancoDAO;
import model.Banco;



public class BancoService {

	BancoDAO cartao = new BancoDAO();

	public List<Banco> listarBancos() {
		return cartao.listarBancos();
	}
}

File: Front_MyWallet/src/app/service/banco.service.ts
import { HttpClient, HttpParams } from '@angular/common/http';
import { ErrorHandler } from '../error.handler';
import { Injectable } from '@angular/core';
import { catchError, Observable } from 'rxjs';
import { environment } from 'src/environments/environment';
import { Banco } from '../models/banco.model';

@Injectable({
  providedIn: 'root'
})
export class BancoService {

  constructor(private http: HttpClient) { }

  getListarBanco(): Observable<Banco>{
    return this.http.get<Banco>(environment.UrlApiBanco + `${"/listarBancos"}`).pipe(catchError(ErrorHandler.handleError));
  }
}
